docker/scylla-sct/ubuntu/Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG SOURCE_IMAGE
+
+FROM ${SOURCE_IMAGE}
+
+ENV USER scylla-test
+
+# Install sudo, collectd and other tools, disable autostart of scylla-server and scylla-housekeeping services,
+# remove the login banner and add a new user.
+#
+# Password for the user is `test' and encrypted using openssl command:
+#    $ echo -n test | openssl passwd -crypt -stdin -salt 00
+#    00hzYw5m.HyAY
+#
+# For more details see man page for useradd(8)
+RUN apt update
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+RUN apt install -y  \
```

Comment:
Can we have it as a single `RUN`? It'll reduce a number of layers in the Docker image.